app/sessionStore.js

```diff
@@ -274,6 +275,11 @@ module.exports.cleanAppData = (data, isShutdown) => {
       })
     }
   }
+  // all data in tabs is transient while we make the transition from window to app state
+  delete data.tabs
+  // if (data.tabs) {
```

Comment:
I left this in here so it was clear how it would work. I wanted to keep all the state management for tabs in on place